addons/xeh/fnc_addClassEventHandler.sqf

```diff
@@ -33,7 +33,7 @@ private _config = configFile >> "CfgVehicles" >> _className;
 
 // init fallback loop when executing on incompatible class for the first time
 if (!GVAR(fallbackRunning) && {ISINCOMP(_className)}) then {
-    diag_log text format ["[XEH]: One or more children of class %1 do not support Extended Event Handlers. Fall back to loop.", configName _config];
+    WARNING_1("One or more children of class %1 do not support Extended Event Handlers. Fall back to loop.", configName _config);
```

Comment:
`[CBA] (xeh) WARNING: One or more children of class All do not support Extended Event Handlers. Fall back to loop. x\cba\addons\xeh\fnc_addClassEventHandler.sqf:36`

is WARNING ok here? it does add the fileLineNumber stuff at the end

Reply:
WARNING requires DEBUG_MODE_NORMAL.

Reply:
`[CBA] (xeh) WARNING: One or more children of class "All" do not support Extended Event Handlers. Fall back to loop.`